includes/templates/single-amp_story.php

```diff
@@ -55,7 +55,7 @@
 
 		<?php
 		// Note that \AMP_Story_Post_Type::filter_frontend_print_styles_array() will limit which styles are printed.
-		wp_styles()->do_items();
+		print_late_styles();
```

Comment:
The `print_late_styles()` will ultimately call `wp_styles()->do_items()` so this makes sense. But I'm curious if there was a specific reason for this change?

Reply:
That was actually something that you originally mentioned in https://github.com/ampproject/amp-wp/issues/1958#issuecomment-473082141, it made sense  since it ultimately calls `do_items()` 😄

Reply:
More readable this way too 👍